views/cms/_form.php

```diff
@@ -73,13 +73,16 @@
 
     <?php echo $form->field($model, 'metaKeywords')->textarea(['rows' => 6]); ?>
 
-    <?php echo $form->field($model, 'status')->dropDownList(CmsStatus::$list); ?>
+    <?php echo $form->field($model, 'status')->dropDownList([
+        Yii::t('modcms', CmsStatus::$$list[CmsStatus::ENABLED]),
+        Yii::t('modcms', CmsStatus::$$list[CmsStatus::DISABLED]),
+    ]); ?>
```

Comment:
We should use listData from BaseEnum https://github.com/yii2mod/yii2-enum/blob/master/helpers/BaseEnum.php#L129
I will update it soon to allow adjustments for message categories but you can use ```yii2mod.enum``` for now.